stl/inc/yvals_core.h

```diff
@@ -176,6 +176,7 @@
 // P0966R1 string::reserve() Should Not Shrink
 // P1001R2 execution::unseq
 // P1006R1 constexpr For pointer_traits<T*>::pointer_to()
+// P1007R3 std::assume_aligned

```

Comment:
Nitpick: We conventionally copy the issue title exactly, so this should follow #47:
```suggestion
// P1007R3 assume_aligned()
```
(The background here is that I "clean up" the various paper titles by simplifying them, making capitalization consistent, removing `std::` qualification, etc. and I like to see that reflected in this list. It doesn't really matter in this case although some papers have very confusing titles that do benefit from this cleanup.)